### PR TITLE
LED matrix circuit issues [MKC-555]

### PR DESCRIPTION
**Pull request by jhansson-ard**
## What This PR Changes
- There were resistors missing in the LED matrix circuit diagram. The resistors keep the matrix from burning, and this PR updates both the circuit diagram and adds a text noting on this.

Original issue: https://github.com/arduino/docs-content/issues/374

**Review:** Compare the old and new image with the issue posed in the issue above.

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Comment by jhansson-ard** [comment]
@per1234 please review to see if you agree with my changes ⭐

File: content/built-in-examples/07.display/RowColumnScanning/RowColumnScanning.md [hunk 1]
```diff
@@ -49,29 +48,26 @@ Here's a matrix of the pin connections, based on the diagram above:
...
 - Arduino Board
-
 - 8 x 8 LED Matrix
-
 - 2 10k ohm potentiometers
-
 - hook-up wires
-
 - breadboard
+- 8 10kΩ resistors
```
Comment (karlsoderby) [inline content/built-in-examples/07.display/RowColumnScanning/RowColumnScanning.md:55]:
```suggestion
- 8x 1k ohm resistors
```

File: content/built-in-examples/07.display/RowColumnScanning/RowColumnScanning.md [hunk 2]
```diff
@@ -49,29 +48,26 @@ Here's a matrix of the pin connections, based on the diagram above:
 ### Hardware Required
 
 - Arduino Board
-
 - 8 x 8 LED Matrix
-
 - 2 10k ohm potentiometers
```
Comment (karlsoderby) [inline content/built-in-examples/07.display/RowColumnScanning/RowColumnScanning.md:52]:
```suggestion
- 2x 10k ohm potentiometers
```

**Review by karlsoderby — APPROVED** [review]
I changed the text to reflect changes as well. I think it looks good now.

**Pull request merged**